webapp/src/index.tsx

```diff
@@ -292,6 +294,18 @@ export default class Plugin {
                     console.log('experimental features disabled');
                     window.localStorage.removeItem('calls_experimental_features');
                 }
+                break;
+            case 'stats':
+                if (!window.callsClient) {
+                    return {error: {message: 'You are not connected to any call'}};
+                }
+                try {
+                    const stats = await window.callsClient.getStats();
+                    console.log(JSON.stringify(stats, null, 2));
+                    return {message: `/call stats "${JSON.stringify(stats)}"`, args};
```

Comment:
Heh, this is clever. I never would have thought to do this. Makes it simpler to display to the user. 👍

Reply:
It's horribly clever :see_no_evil: . To be honest I wanted a way to display an ephemeral message directly from the client but to my surprise it doesn't look like possible. The solution is hacky but it gets the job done so why not.